code/go/0chain.net/smartcontract/stakepool/stakepool.go

```diff
@@ -678,6 +668,35 @@ func StakePoolLock(t *transaction.Transaction, input []byte, balances cstate.Sta
 	return out, err
 }
 
+func validateLockRequest(t *transaction.Transaction, sp AbstractStakePool, err error) (string, error) {
```

Comment:
We don't need the third param `err error`, let's remove it to fix the lint errors

Reply:
lol, dumb thing, it was introduced by goland refactoring tool when extracting func

Reply:
Resolved